tests/test_wps_generate_climos.py

```diff
@@ -3,21 +3,21 @@
 from pywps import Service
 from pywps.tests import assert_response_success
 
-from .common import client_for, TESTDATA
+from .common import TESTDATA, run_wps_process
 from thunderbird.processes.wps_generate_climos import GenerateClimos
 
 
-def run_wps_generate_climos(netcdf, datainputs):
-    client = client_for(Service(processes=[GenerateClimos()]))
-
-    resp = client.get(
-        service="wps",
-        request="Execute",
-        version="1.0.0",
-        identifier="generate_climos",
-        datainputs=datainputs,
-    )
-    assert_response_success(resp)
+def parameters(netcdf, kwargs):
+    return (
+        "netcdf=@xlink:href={0};"
+        "operation={operation};"
+        "climo={climo};"
+        "resolutions={resolutions};"
+        "convert_longitudes={convert_longitudes};"
+        "split_vars={split_vars};"
+        "split_intervals={split_intervals};"
+        "dry_run={dry_run};"
+    ).format(netcdf, **kwargs)
```

Comment:
Nice! Only suggestion would be to rename the method, `get_parameters` or `build_params`, or something along those lines.